CONTRIBUTING.md

```diff
@@ -10,12 +10,17 @@ https://www.vagrantup.com/downloads.html
 ### 3 Capture the powers of vagrant
   * In the repo dir: <code>vagrant up</code> (Safely ignore: 'dpkg-preconfigure: unable to re-open stdin: No such file or directory')
   * If changes have been made since running vagrant up: <code>vagrant provision</code>
-  * To login to the machine: <code>vagrant ssh</code>
-  * `cd  /vagrant/` to navigate to the refuge repo.
-  * There is a rake wrapper to execute commands in refuge repo on the vagrant machine
-  without using ssh. Run `rake vagrant:shell[command]`
-  * To start the rails server bind it to the vagrant machine's IP address after running `vagrant ssh`, use: `rails s -b 0.0.0.0`. Otherwise, run: `rake vagrant:shell['rails s -b 0.0.0.0']`
-  * The website is available at `localhost:3000`
+  
+  Please note that there are two ways to run commands in vagrant:
+  1. A local `rake` wrapper that allows direct execution on the machine.
```

Comment:
For some reason this isn't rendering as a numbered list. It may have to be indented to recognize a change in list type